rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -1076,6 +1103,7 @@ def _load_model_class(
         label_data: RasaModelData,
         entity_tag_specs: List[EntityTagSpec],
         meta: Dict[Text, Any],
+        finetune_mode: bool,
     ) -> "RasaModel":
 
         return cls.model_class().load(
```

Comment:
?
```suggestion
        return cls.load(
```

Reply:
Ahh no, `model_class()` is needed to get the correct architecture type, `DIET`, `DIET2BOW`, `DIET2DIET` and call `load` on top of it.

Reply:
Mhm, shouldn't that still work 🤔  `cls` will always be the things it's called upon. means `DIET2Bow.class_function` will have the `cls` of `DIET2Bow`

Reply:
`cls` here is `DIETClassifier`. `DIET, DIET2BOW, DIET2DIET` are classes for different model architectures.

Reply:
`cls` here is `DIETClassifier`. `DIET, DIET2BOW, DIET2DIET` are classes for different model architectures.

Reply:
mhm, not sure what you mean. I'll trust you 👍